P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+        <item>communication technologies such as chats, messengers, or online forums;</item>
+        <item>social media platforms and applications such as X (the platform formerly known as
+          Twitter), Facebook, Instagram, or WhatsApp;</item>
+        <item>the communication functions of collaborative platforms and projects (e.g. an online
+          learning environment, or a Wikipedia <soCalled>talk</soCalled> page);</item>
+        <item>3D environments (e.g. Second Life or gaming environments);</item>
+        <item>or similar technologies and services on the internet.</item>
+      </list></p>
+    <p>CMC is, by default, multimodal. Whereas early CMC systems
+      (e.g. Internet Relay Chat, <soCalled>IRC</soCalled> for short, the Usenet
+      <soCalled>newsgroups</soCalled>, or even the Unix <name>talk</name> system)
+      were completely ASCII-based, the lion’s share of contemporary CMC technologies and
+      applications allow for the combination of different semiotic resources (e.g. written or spoken
+      language with graphic icons and images) and even for the combined use of different CMC
+      technologies on one platform (e.g. combined use of an audio connection, a chat system, and a
+      3D interface in which users control a virtual avatar as in many multiplayer online computer
+      games or in virtual worlds such as Second Life).</p>
+  </div>
+  <div type="div2" xml:id="CMCUnits">
+    <head>Basic Units of CMC</head>
+    <p>This section describes the encoding mechanisms for the basic units of CMC and for their
+      combined use to encode CMC data.</p>
+    <p>We refer to units which are produced by an interlocutor to contribute to an ongoing CMC
+      interaction or joint CMC activity as <emph rend="italic">basic CMC units</emph>. Contributions
+      to an ongoing interaction are produced to perform a move as part of the further development of
+      the interactional sequence, for instance in chats or forum discussions. Contributions to joint
+      CMC activities may not all be directly interactional; some may be part of a collaborative
+      project of the involved individuals. For example, editing activities in a shared text editor
+      or whiteboard in parallel with an ongoing CMC interaction (chat, audio conversation, or
+      audiovideo conference) in the same CMC environment in which these editing activities are
+      discussed by the collaborators.</p>
+    <p>Basic units of CMC can be described according to three criteria: (i) the temporal properties
+      of how these contributions are produced by their creators, transmitted via CMC systems, and
+      made accessible for the recipients; (ii) whether the unit as a whole is realised in a verbal
+      or nonverbal mode; (iii) for verbal units: whether the unit is realised in the written or
+      spoken mode. A taxonomy of basic CMC units resulting from these criteria is given in Fig.
```

Comment:
I fixed the redundant “in”.
What to do about the Stylesheets not processing `head/ptr` here?
I think one easy path forward is to remove the `<ptr>` and provide that information some other way. Perhaps:
~~~xml
A taxonomy of basic CMC units resulting from these criteria is given in
<ptr type="cit" target="#BIB_CMC_Core"/>, and reproduced here in
<ptr target="#cmcunits-taxonomy"/>.</p>
    <figure xml:id="cmcunits-taxonomy">
      <graphic url="Images/cmcunits-taxonomy.png"/>
      <head>Taxonomy of basic CMC units</head>
    </figure>
~~~

Reply:
I don't think it needs any kind of link since the figure immediately follows the sentence. We don't provide links to figures when we mention them in the preceding sentence in other chapters (I checked), so I repaired the problem by removing the problematic `<ptr>` here: https://github.com/TEIC/TEI/pull/2537/commits/5bdda679adcae9954d01398bcfc2bb0483ea70ff

Reply:
Applied my repair in https://github.com/TEIC/TEI/pull/2537/commits/7ca18f41bc4f8544b249cfff84478c1003bdb634